roles/openshift_logging/tasks/start_cluster.yaml

```diff
@@ -36,7 +36,7 @@
     name: "{{ object }}"
     namespace: "{{openshift_logging_namespace}}"
     replicas: "{{ openshift_logging_mux_replica_count | default (1) }}"
-  with_items: "{{ mux_dc.results.results[0]['items'] if 'results' in mux_dc else {} | map(attribute='metadata.name') | list }}"
+  with_items: "{{ mux_dc.results.results[0]['items'] | map(attribute='metadata.name') | list if 'results' in mux_dc else [] }}"
```

Comment:
`mux_dc.results.results[0]['items'] if 'results' in mux_dc else {} | map(attribute='metadata.name') | list`
In this case, the `if` is before the `map(...)`.  What this does is set `object` to `mux_dc.results.results[0]['items']`, which is the entire definition of the dc, which is passed to the oc scale command line, which causes a failure.

`mux_dc.results.results[0]['items'] | map(attribute='metadata.name') | list if 'results' in mux_dc else []`
In this case, the `if` is after the `map(...)`.  What this does is set `object` to the evaluation of the entire pipeline `mux_dc.results.results[0]['items'] | map(attribute='metadata.name') | list`, which is the list of dc names, which is the argument expected by oc scale.